parser/transaction_test.go

```diff
@@ -85,9 +84,8 @@ func TestV5TransactionParser(t *testing.T) {
 		if len(rest) != 0 {
 			t.Fatalf("Test did not consume entire buffer, %d remaining", len(rest))
 		}
-		if bytes.Equal(tx.cachedTxID, []byte(txtestdata.Txid)) {
-			t.Fatal("txid")
-		}
+		// Currently, we can't check the txid because we get that from
+		// zcashd (getblock rpc) rather than computing it ourselves.
```

Comment:
Add an issue reference here so we remember to re-add this test when fixing the issue:
```suggestion
		// Currently, we can't check the txid because we get that from
		// zcashd (getblock rpc) rather than computing it ourselves.
		// https://github.com/zcash/lightwalletd/issues/392
```